src/pybind/mgr/dashboard/services/rgw_client.py

```diff
@@ -247,7 +247,7 @@ def is_service_online(self, request=None):
         Consider the service as online if the response contains the
         specified keys. Nothing more is checked here.
         """
-        request({'format': 'json'})
+        _ = request({'format': 'json'})
```

Comment:
What's that for?

Reply:
To prevent a lint error. Request returns a value, so it must be processed, otherwise you'll get a lint warning.